### PR TITLE
Add Information About OPENLINEAGE_NAMESPACE

### LINKED ISSUE #1607: More Clear usage and impact of OPENLINEAGE_NAMESPACE
**Affected Document:**
https://docs.astronomer.io/astro/set-up-data-lineage

**Problem Type (Priority):**
- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [x] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**
In the documentation, there is a mention on setting `OPENLIEAGE_NAMESPACE` for integrating data lineage (e.g. spark). However, there isn't too much of information on what it does and what would be the impact of it in the lineage data.

since OPENLINEAGE_NAMESPACE is defined by default as deployment namespace of Astro for the JOB, and dataset specific for the datasets, we should first mention that so that the users would understand what it is, and also understand what value to correctly set it up (e.g. where to find the current namespace, and set it up with it accordingly). Also, the document should let the user know the impact of setting this value - for example, setting the openlineage namespace arbitrarily may impact how the graph will display your jobs - or datasets, etc.)

**Additional Notes:**
- https://docs.astronomer.io/astro/data-lineage#view-the-lineage-graph-for-a-data-pipeline

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1607

### CODE REVIEW

**Review by howardyoo — COMMENTED** [review]
openlineage-namespace -> OPENLINEAGE_NAMESPACE (capital letters)
I kind of feel this is just too simple.

**Comment by kwcanuck** [comment]
@howardyoo feel free t provide your edits and suggestions to expand the definition. Also, where do users find OPENLINEAGE_NAMESPACE information?

**Comment by howardyoo** [comment]
_Here's the detailed paragraphs_

When integrating an external data lineage system with Astro, or [you are running Astro dev locally](https://docs.astronomer.io/astro/create-project), you are required to define the `OPENLINEAGE_NAMESPACE` environment variable, along with other variables like `OPENLINEAGE_URL` and `OPENLINEAGE_API_KEY`. 

Normally, these values will be pre-populated when you are running DAG in Astro, so users do not have to worry about it. However, for external systems, they are not automatically avaialble and thus must be set correctly for the OpenLineage events to be emitted correctly.

- OPENLINEAGE_NAMESPACE: You can find it in Astro UI's Deployments page, right next to the deployment name in a format of `<word>-<word>-<four-digit-number>` (e.g. universal-orbit-2141).
- OPENLINEAGE_URL : Should be set as `https://astro-<your-astro-base-domain>.datakin.com`
- OPENLINEAGE_API_KEY : Can be found in `https://cloud.<your-astro-base-domain>.io/settings`. Look for `Lineage API Key`.

**Review by howardyoo — CHANGES_REQUESTED** [review]
copy the value that includes a four-digit number <-- should be copy the value that is in the format of `<word>-<word>-<four-digit-number>` (e.g. `some-thing-3121`) I think people may get confused, if they do not recognize this format, so we need to be clear about it.

**Comment by kwcanuck** [comment]
@howardyoo I have modified the text. Please review when time permits.

**Comment by howardyoo** [comment]
Hi @kwcanuck , I just noticed this when reviewing the change:
image
Will it be okay for you to merge these two sections together? For example, the instruction at the first section covers how to obtain the Openlineage API key which is redundant to the last instruction of obtaining api key. I believe we should merge them.

**Comment by kwcanuck** [comment]
@howardyoo good catch. I have removed the duplicated content. Back to you for review.

**Review by howardyoo — APPROVED** [review]
LGTM! Thanks for the work!

**Comment by kwcanuck** [comment]
@jwitz ready for peer review.

File: astro/set-up-data-lineage.md [hunk 1]
```diff
@@ -29,16 +29,7 @@ The data lineage graph in the Cloud UI shows lineage data that is emitted with b
...
-- Specify your OpenLineage API key in the external system. See the following integration guides for specific instructions.
-
-### Retrieve an OpenLineage API key
-
-1. In the Cloud UI, click **Settings**.
-2. Copy the value in the **Lineage API Key** field.
-3. Specify your Organization's OpenLineage API key in the external system's configuration.
+When you integrate an external data lineage system with Astro or you are working with Astro locally, you need to define the `OPENLINEAGE_NAMESPACE`, `OPENLINEAGE_URL`, and `OPENLINEAGE_API_KEY` environment variables to make sure the correct OpenLineage data is sent to Astro. To locate the `OPENLINEAGE_NAMESPACE` variable in the Cloud UI, select a Workspace and then copy the value with the format `<text>-<text>-<four-digit-number>` next to the Deployment name. The format for the `OPENLINEAGE_URL` variable is `https://astro-<your-astro-base-domain>.datakin.com`. To locate the `OPENLINEAGE_API_KEY` variable, go to `https://cloud.<your-astro-base-domain>.io/settings` and copy the value in the **Lineage API Key** field.
```
Comment (jwitz) [inline astro/set-up-data-lineage.md:32]:
```suggestion
When you integrate an external data lineage system with Astro or you are working with Astro locally outside of a supported Airflow operator, you need to provide a Deployment namespace, your Organization's OpenLineage URL, and your organization's OpenLineage API key. This information is used to send OpenLineage data to the correct place in Astro. 

To locate the your Deployment namespace in the Cloud UI, select a Workspace and then copy the value with the format `<text>-<text>-<four-digit-number>` next to the Deployment name. To locate your Organization's OpenLineage URL and OpenLineage API key, go to `https://cloud.<your-astro-base-domain>.io/settings` and copy the values in **OpenLineage URL** and **OpenLineage API key** respectively.
```

Comment (jwitz) [reply astro/set-up-data-lineage.md:32]: - Updated this to use the UI for copying the OpenLineage URL
- Removed environment variable formatting as there's no actual environment variable you need to configure with these names. In the setups that follow, you provide the value as opposed to a specific env var.
- Made it clear that this is only for integrating with an external system _that doesn't already have a supported Airflow operator_.

**Review by jwitz — APPROVED** [review]

**Pull request merged**